src/domdiv/tools/new_cards.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+# coding=utf-8
+
+import csv
+import sys
+import json
+
+new_cards_db = []
+new_cards_en_us = {}
+
+with open(sys.argv[1]) as csvfile:
```

Comment:
Let's use argparse, even for something this small?

Reply:
> Looks like the tests need to be updated with the new card number?

Fixed

> I also have the concern that the non-JSON files with get out of date with the JSON files and be hard to reconcile. Are you thinking the CSVs should become source of truth? Otherwise there should probably be checks to make sure the scripts aren't duplicating cards, etc.

No, the JSON is still the source of truth.  This was really just a quick way to enter a bunch of new cards.  From past experience, it was probably 3x or more faster.  It allowed me to focus on the content and not deal with JSON formatting up front.   In the past, when editing straight JSON, there was a tendency to get lost in the trees.  And if something needed changed in several places, it was harder to find.
But once done, it was done.   In fact, I think I tweeked the JSON in a place or two.  So I don't view the csv as something that is kept up to date.  This is more of a one time use to make a bunch of edits easier and faster.

I really did not intend to spend too much time on it and make it too bullet proof.  The same problem of duplicate cards can happen when editing straight JSON (In fact, I have had that happen in the past when I cut-n-pasted JSON to enter cards.) I really only uploaded in case it would be helpful to someone in the future.

Reply:
Ok, that's fine.